src/integrationTest/codelens.js.test.ts

```diff
@@ -15,7 +15,7 @@ const CODELENS_TEST_TIMEOUT_MILLIS = 10000
 
 const workspaceFolder = getTestWorkspaceFolder()
 
-describe('SAM Local CodeLenses (JS)', async () => {
+describe('SAM Local (JS)', async () => {
```

Comment:
What is the motivation for this change? Seeing a test print out "SAM Local (JS)" implies it is exercising a full flow, where this test is exercising that CodeLenses appear.

Reply:
originally thought these were testing SAM functionality. But they are not, so I will remove "SAM" from the title and restore "CodeLens"